e2e-projects/sveltekit/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "sveltekit",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+		"lint": "prettier --plugin-search-dir . --check . && eslint .",
+		"format": "prettier --plugin-search-dir . --write .",
+		"slicemachine": "start-slicemachine",
+		"slicemachine:dev": "NODE_ENV=development SM_ENV=staging start-slicemachine"
+	},
+	"devDependencies": {
+		"@slicemachine/adapter-sveltekit": "workspace:*",
```

Comment:
In other comments, I suggested we stick to `create-svelte`'s default template, but I could see us diverging from dependency versions since we would have some instability using `^` versions.

Do you think we should change the dependencies in the e2e project to exact versions?